README.md

```diff
@@ -171,14 +171,17 @@ search.addWidget(
 
 ```js
 /**
- * Instantiate a list of refinement based on a facet
+ * Instantiate a list of refinements based on a facet
  * @param  {String|DOMElement} options.container Valid CSS Selector as a string or DOMElement
  * @param  {String} options.facetName Name of the attribute for faceting
  * @param  {String} options.operator How to apply refinements. Possible values: `or`, `and`
  * @param  {String[]} [options.sortBy=['count:desc']] How to sort refinements. Possible values: `count|isRefined|name:asc|desc`
  * @param  {String} [options.limit=100] How much facet values to get.
- * @param  {String|String[]} [options.cssClass=null] CSS class(es) for the main `<ul>` wrapper.
+ * @param  {String|String[]} [options.rootClass=null] CSS class(es) for the root `<ul>` element
```

Comment:
Actually since I was able to use ul li (after re-trying), we do not need anymore div div